exercism/bob/bob.js

```diff
@@ -0,0 +1,14 @@
+var Bob = function() {};
+
+Bob.prototype.hey = function(text) {
+  if (text.match(/[A-Z]/) && text === text.toUpperCase()) {
```

Comment:
One of these comparisons seems redundant to me....
Your checking all uppercase letter with a regex but then also checking that all the letters are also uppercased? Why?